mkosi/util.py

```diff
@@ -304,3 +304,14 @@ def qemu_check_vsock_support(log: bool) -> bool:
         raise e
 
     return True
+
+
+def format_bytes(num_bytes: int) -> str:
+    if num_bytes >= 1024 * 1024 * 1024:
+        return f"{num_bytes/1024**3 :0.1f}G"
+    if num_bytes >= 1024 * 1024:
```

Comment:
When you're moving this, could you check whether 
```suggestion
    if num_bytes >= 1024**3:
        return f"{num_bytes/1024**3 :0.1f}G"
    if num_bytes >= 1024**2:
```
type checks nowadays? The only reason it's written like this was that mypy was unable to figure out that `1024**3` is an int, back when this was last touched.